src/js/modules/accounts/components/forms/EditUserForm.js

```diff
@@ -99,7 +67,8 @@ const EditUserForm = ({ classes, handleSubmit, submitting, status }) => {
                   <Field
                     name="lastName"
                     type="text"
-                    component={renderField}
+                    autoComplete="off"
```

Comment:
Not sure this is a good piece of design. Having to specify autoComplete on or off for every component seems to violate the goal of one usage per component. I'd prefer just a separate autoComplete component, or maybe an HOC for rendering a field

Reply:
The autoComplete input attribute helps password managers infer the purpose of a field in a form. Proper usage of autoCompletes means better UX. "On" and "off" weren't keywords I chose. <https://www.chromium.org/developers/design-documents/create-amazing-password-forms>

In this case, I don't want browser password managers suggesting values for the `EditUserForm` because the form is obviously supposed to take values that were not the user's original e-mail/name/etc.

Reply:
Oh, my bad. If it's a browser thing then keep. Thought it was a prop you designed.